x-pack/solutions/security/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -314,7 +317,7 @@ export const TimelineFlyout = memo(() => {
     <ExpandableFlyout
       registeredPanels={expandableFlyoutDocumentsPanels}
       paddingSize="none"
-      customStyles={{ 'z-index': (euiTheme.levels.flyout as number) + 2 }}
+      customStyles={{ 'z-index': (euiTheme.levels.flyout as number) + 3 }}
```

Comment:
Nit: since these z-indexes are all related to each other, we should try to capture that relationship in some way. Maybe a wrapping hook? E.g. :

```ts
const { timelineFlyoutZIndex, exceptionFlyoutZIndex } = useFlyoutZIndexes();
```